### PR TITLE
[onnx]logsoftmax/tanh/exp/flatten/sqrt/log/hardsigmoid

### PR DESCRIPTION
**Pull request by xiaxin1998**

### CODE REVIEW

File: pyzoo/zoo/pipeline/api/onnx/mapper/tanh.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from zoo.pipeline.api.onnx.onnx_helper import OnnxHelper
+import zoo.pipeline.api.keras.layers as zlayers
+
+
+class TanhMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(TanhMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        relu = zlayers.Activation("tanh")
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/tanh.py]: relu?

File: pyzoo/zoo/pipeline/api/onnx/mapper/hardsigmoid.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+from zoo.pipeline.api.onnx.onnx_helper import OnnxHelper
+import zoo.pipeline.api.keras.layers as zlayers
+
+
+class HardSigmoidMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(HardSigmoidMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        log = zlayers.Activation("hard_sigmoid")
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/hardsigmoid.py]: log?

File: pyzoo/zoo/pipeline/api/onnx/mapper/flatten.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from zoo.pipeline.api.onnx.onnx_helper import OnnxHelper
+import zoo.pipeline.api.keras.layers as zlayers
+
+
+class FlattenMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(FlattenMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        relu = zlayers.Flatten()
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/flatten.py]: relu?

File: pyzoo/zoo/pipeline/api/onnx/mapper/exp.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+from zoo.pipeline.api.onnx.onnx_helper import OnnxHelper
+import zoo.pipeline.api.keras.layers as zlayers
+
+
+class ExpMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(ExpMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        relu = zlayers.Exp()
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/exp.py]: name?

**Comment by xiaxin1998** [comment]
18888/view/ZOO-PR/job/ZOO-PR-Validation/1074/  Jenkins passed @zhichao-li

**Pull request merged**